src/js/extension.js

```diff
@@ -66,8 +66,124 @@ var Extension;
             // has a .base value pointing to the editor
             // owning us. has been given a .name if no
             // name present
-        }
-        // core API definition for an "extension" follows:
+        },
+
+        /* parent: [boolean]
+         *
+         * setting this to true will set the .base property
+         * of the extension to be a reference to the
+         * medium-editor instance that is using the extension
+         */
+        parent: false,
```

Comment:
Yeah, I agree.  I haven't seen any extensions which haven't needed this property.  Maybe we should just do away with it and always set `.base` on all extensions?